adapters/adapter_test_util.go

```diff
@@ -0,0 +1,69 @@
+package adapters
+
+import (
+	"fmt"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/mxmCherry/openrtb"
+	"net/http"
+)
+
+// OrtbMockService Represents a scaffolded OpenRTB service.
+type OrtbMockService struct {
+	Server          *httptest.Server
+	LastBidRequest  *openrtb.BidRequest
+	LastHttpRequest *http.Request
+}
+
+// BidOnTags Produces a map of TagIds, based on a comma separated strings. The map
+// contains the list of tags to bid on.
+func BidOnTags(tags string) map[string]bool {
+	values := strings.Split(tags, ",")
+	set := make(map[string]bool)
+	for _, tag := range values {
+		set[tag] = true
+	}
+	return set
+}
+
+// SampleBid Produces a sample bid based on params given.
+func SampleBid(width *uint64, height *uint64, impId string, index int) openrtb.Bid {
+	return openrtb.Bid{
+		ID:    "Bid-123",
+		ImpID: fmt.Sprintf("div-adunit-%d", index),
+		Price: 2.1,
+		AdM:   "<div>This is an Ad</div>",
+		CrID:  "Cr-234",
+		W:     *width,
+		H:     *height,
+	}
+}
+
+// VerifyStringValue Helper function to assert string equals.
```

Comment:
Let's follow the convention used in the Go core libs (`http` and `httptest`) here, and move these into the `adapterstest` package.